docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,129 @@
+# Tutorial
+
+### Prerequisites
+
+To deploy wordpress-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the environment
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up another environment by adding a new model with the following command.
```

Comment:
Perhaps remove the reference to the environment and just focus on having another model? A model is basically the same as an environment so this could reduce duplication

Reply:
Changed "environment" to "model", i.e.
```
### Setting up the tutorial model

To easily clean up the resources and to separate your workload from the contents of this tutorial,
it is recommended to set up a new model with the following command.
```

Thank you for the suggestion!